src/StepDefinition/MinkElementTrait.php

```diff
@@ -195,7 +195,7 @@ public function textAppearsTimesInElement($text, $element, $number, $selector =
    *
    * @throws \Behat\Mink\Exception\ExpectationException
    */
-  public function iWaitForElementToAppear($waitTime = 5000, $num, $element) {
+  public function iWaitForElementToAppear($num, $element, $waitTime = 5000) {
```

Comment:
if behat stopped working automatically assigning the right arguments but just applying the order, this cannot work both spec definitions any more. let's make sure buth work and let's add another wrapping/forwarding method if necessary so we can have custom order for each of the two step definitions

Reply:
well it does assign the right values (now) otherwise behat would fail (we're sometimes using it with and sometimes wo/ w8 time).

But I was also testing with changing the annotation to a simpler one and it worked as well, but this was the least amount of changes to make the behat work.